drivers/gpio/Kconfig

```diff
@@ -37,6 +37,13 @@ config GPIO_GET_DIRECTION
 	  via gpio_port_get_direction(), gpio_pin_is_input(), and
 	  gpio_pin_is_output().
 
+config GPIO_GET_CONFIG
+	bool "Support for get configuration function"
```

Comment:
This should have `[EXPERIMENTAL]` listed in the prompt, and also `select EXPERIMENTAL`.

Reply:
Done